charts/tikv-importer/templates/tikv-importer-service.yaml

```diff
@@ -5,15 +5,16 @@ metadata:
   labels:
     app.kubernetes.io/name: {{ template "chart.name" . }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/instance: {{ .Values.clusterName }}
     app.kubernetes.io/component: importer
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
 spec:
-  clusterIP: None
```

Comment:
We should keep this, why remove it here?
This service is a headless service through which Prometheus can access the Pod via Pod name.

Reply:
reverted.